notes-docs.md

```diff
@@ -0,0 +1,51 @@
+Philosophy: one thing to do things
+
+Questions:
+
+-   What to do about the egghead lessons? They have a reference the decorator
+    syntax.
+
+-   The README is rather long. Credits, testimonials, and sponsors in multiple
+    places.
+
+-   How to name "compatible property tracking" as opposed to proxy-based?
+    How does one configure it, or it is an automatic fallback?
+
+-   What to do about translations? They will be out of date.
+
+-   Is there a way to regenerate assets? flow.png for instance uses
+    decorators and needs to be updated.
+
+-   Any code sandbox links need to be updated. jsfiddle as well.
```

Comment:
Feel free to do that. There is nothing wrong with having CS links that are on your personal account. Unless of course, you have a free account with a limit or other reasons. We can figure that out eventually.

Reply:
I have a paid account, maybe I can share a space or something. Will try to figure that out

Reply:
I will wait until clarity concerning codesandbox and jsfiddle before I start messing around with those. Do we want to consolidate to a single system (only codesandbox or jsfiddle) or do they serve different purposes?

Reply:
I think it would be nice to have all examples neatly tucked in a single platform. Even though jsfiddle is generally faster and approachable, it uses UMD format which I believe shouldn't be used for any serious work.

Anyway, all interactive examples will need some alpha release first to make that work, so we can leave that for the last.

Reply:
I think it is historical indeed, and I think we can put everything on CodeSandbox